docs/tutorials/guides/hello-velaptor.mdx

```diff
@@ -0,0 +1,291 @@
+The 4 different default font types are:
+- TimesNewRoman-Regular
+- TimesNewRoman-Bold
+- TimesNewRoman-Italic
+- TimesNewRoman-BoldItalic
+
+<!-- TODO: Need to create a link to the guide once it is created.-->
+Default font types are the 4 types of fonts that come _**built-into**_ Velaptor. For more info, refer to the font guide.
+:::
+
+
+#### <span class="color-sub-step">Step 5.4</span>
+
+Now that we have a renderer and a font to render text to the screen, we need to create a method override of the `Window.OnDraw()` method.
+The `OnDraw()` method is part of the game loop.
+
+```csharp
+protected override void OnLoad()
+{
+    ...
+    base.OnLoad();
+}
+
+// Create the new method override for rendering to the screen
+protected override void OnDraw(FrameTime frameTime)
+{
+    base.OnDraw(frameTime);
+}
+```  
+
+#### <span class="color-sub-step">Step 5.5</span>
+
+Next, we need to use the renderer to render some text to the screen using our font.  Add the code below to the `OnDraw()` method to accomplish this.
+
+```csharp
+// Draws to the screen.  Executes one time for every iteration of the game loop
+// and always AFTER the OnUpdate() method has finished.
+protected override void OnDraw(FrameTime frameTime)
+{
+    var x = (int)(Width / 2); // Get the center of the window horizontallly 👈🏼
+    var y = (int)(Height / 2); // Get the center of the window vertically 👈🏼
+
+    IRenderer.Begin(); // This must be called before we can render anything 👈🏼
+
+    // Render the string to the screen with the color orange 👇🏻
+    this.fontRenderer.Render(this.font, "Hello Velaptor!!", x, y, Color.Orange);
+
+    IRenderer.End(); // This must be called to end the rendering process 👈🏼
+
+    base.OnDraw(frameTime);
+}
+```
+
+:::note
+Some game engines and frameworks use the top left corner of an object when rendering at a particular position.  Velaptor does not work like this.
+When rendering text, the position of the text is relative to the center of the entire text or texture content.
+
+What this means is if you render the text at position _**(0,0)**_, you would only be able to see the bottom right quadrant of the text because
+the other 3 quadrants would be rendered off-screen!!  Make sure to take this into account when it comes to text and object render positioning.
+<img style={{width: '75%'}} src={require('@site/static/img/tutorials/guides/hello-velaptor/text-origin.png').default} alt="Example banner"/>
+:::
+
+#### <span class="color-sub-step">Step 5.6</span>
+
+Now the best part!! Let's see the fruits of your labor and run the application.
+You should see the result below!! 🥳
+<img style={{width: '75%'}} src={require('@site/static/img/tutorials/guides/hello-velaptor/hello-velaptor-window.jpg').default} alt="Example banner"/>
+
+
+### <span class="color-step">Step 6: Something to update</span>
+
+A game cannot be a game without adding interactivity and updating something to make things happen.  Otherwise, you are just looking at a still picture!
```

Comment:
What is an example of the "something"? Can it be deleted?
```suggestion
A game cannot be a game without adding interactivity and updating something to make things happen.  Otherwise, you are just looking at a still picture!
```

Reply:
The _**something**_ all depends on what the person is creating.  There is no way for me to know that.
I can change the word _**something**_ to **the game**.  Like below.

```suggestion
A game cannot be a game without adding interactivity and updating the game to make things happen.  Otherwise, you are just looking at a still picture!
```

Reply:
I see what you're saying.  I feel like you can delete that part of the sentence, then.  End the first sentence with interactivity and the next sentence refers to looking at a still picture w/out the interactivity.  Thoughts?

Reply:
Like this?

```suggestion
A game cannot be a game without updating the game to make things happen.  Otherwise, you are just looking at a still picture with no interactivity!
```

Reply:
How about replacing "the game" with it?  I don't like that whole section, but it is what it is.